Common/Product/SharedProject/Automation/VSProject/OAReferences.cs

```diff
@@ -88,7 +88,7 @@ private Reference FindByName(string stringIndex)
         public Reference Add(string bstrPath)
         {
             // ignore requests from the designer which are framework assemblies and start w/ a *.
-            if (string.IsNullOrEmpty(bstrPath) || bstrPath.StartsWith("*"))
+            if (string.IsNullOrEmpty(bstrPath) || bstrPath.StartsWith("*", StringComparison.Ordinal))
```

Comment:
`StartsWithOrdinal`

Reply:
Files under `Common` directory don't have access to things we declare in `Microsoft.PythonTools` namespace.